homeassistant/components/idasen_desk/quality_scale.yaml

```diff
@@ -0,0 +1,92 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration does not use polling.
+  brands: done
+  common-modules:
+    status: todo
+    comment: |
+      The cover and sensor entities could move common initialization to a base entity class.
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  docs-high-level-description: todo
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      This integration does not provide configuration parameters.
+  docs-installation-parameters:
```

Comment:
It is not configurable and comes only from discovery:
![image](https://github.com/user-attachments/assets/4ccec12c-6b20-49eb-81d0-678c991db683)

Reply:
We have other integrations doing that. Otherwise the user would not be able to add the integration from the menu.
Note that discovery is not done on the user step. Only the result of discovery is used.

Reply:
what do you mean with "available (Bluetooth) devices"?

Reply:
something like that 
![image](https://github.com/user-attachments/assets/ef71d43b-5cde-4db8-8bb7-2a017f5b8e4c)

Reply:
maybe I am missing something, but why would we show devices that are not the desk?
we already show all the discovered desks and the user can choose one: https://github.com/home-assistant/core/pull/132368#discussion_r1871809390 (In my case, I only have 1 to choose)